readme.md

```diff
@@ -153,6 +153,21 @@ This repository contains [provisioning service client SDK](./provisioning/servic
 
 > Device Explorer is no longer supported. A replacement tool can be found [here](https://github.com/Azure/azure-iot-explorer).
 
+## Certificates -  Important to know
+
+The Azure IoT Hub certificates presented during TLS negotiation shall be always validated using the appropriate root CA certificate(s).
+
+Always prefer using the local system's Trusted Root Certificate Authority store instead of hardcoding the certificates (i.e. using certs.c such as our samples require in certain combinations). 
```

Comment:
"i.e. using certs.c such as our samples require in certain combinations" is not relevant to us